letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -530,7 +542,19 @@ CheckPathPermissions() {
 UNLIKELY_EOF
 }
 
-if [ "$1" = "--le-auto-phase2" ]; then
+if [ "$1" = "--le-auto-phase2" -a "$DEPRECATED_OS" = 1 ]; then
+  # Phase 2 damage control mode for deprecated OSes.
+  # In this situation, we bypass any bootstrap or certbot venv setup.
+  shift 1
+
+  error "Your system is not supported by certbot-auto anymore, and so certbot cannot be installed or upgraded."
```

Comment:
What do you think about making the rest of the code in this case look something like:
```
  error "Your system is no longer supported by certbot-auto."
  if [ -f "$VENV_BIN/letsencrypt" ]; then 
    error "Certbot will no longer receive updates."
    error "Please visit https://certbot.eff.org/ to check for other alternatives."
    "$VENV_BIN/letsencrypt" "$@" 
    exit 0
  else 
    error "Certbot cannot be installed."
    error "Please visit https://certbot.eff.org/ to check for other alternatives."
    exit 1
  fi
```

Reply:
Ah. With your previous comment, I was hesitating on the exit code value depending on the situation (certbot installed or not). Yes, it makes sense, and I add that.